### PR TITLE
Add Secure File download location information

### PR DESCRIPTION
**Pull request by alexkokkinos**
Adds information about where the secure file is downloaded.

### CODE REVIEW

File: docs/build-release/tasks/utility/download-secure-file.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ monikerRange: 'vsts'
 
 Use this task to download a [secure file](../../concepts/library/secure-files.md) from the server during a build or release.
```
Comment (madhurig) [inline docs/build-release/tasks/utility/download-secure-file.md]: Can you also add a note that the full path to the downloaded file is accessible via $env:DOWNLOADSECUREFILE_SECUREFILEPATH. If multiple versions of the task are used the variables are disambiguated as DOWNLOADSECUREFILE1_SECUREFILEPATH, DOWNLOADSECUREFILE2_SECUREFILEPATH ...

Comment (alexkokkinos) [reply docs/build-release/tasks/utility/download-secure-file.md]: Sure, I can do that!

**Comment by steved0x** [comment]
@alexkokkinos @madhurig Thank you for your contribution and review, I am merging this now and it should be live soon. Thank you!

**Comment by PatMyron** [comment]
Thanks for adding this! The powershell syntax was a bit confusing as someone unfamiliar with powershell. Could we make the syntax less powershell specific?

**Pull request merged**